packages/dds/tree/src/simple-tree/proxies.ts

```diff
@@ -242,6 +255,14 @@ export function createObjectProxy<TSchema extends FlexObjectNodeSchema>(
 	return proxy;
 }
 
+function getAllowedTypes(schema: ImplicitFieldSchema): AllowedTypes {
+	return normalizeAllowedTypes(schema instanceof FieldSchema ? schema.allowedTypes : schema);
```

Comment:
@CraigMacomber I ended up having to repeat this pattern in a few places. Do we already have a utility for this that I can use?

Reply:
I don't think we do. I think normalizing things once and reusing it is likley better than normalizing when needed, especially given that some users of schema will likely want  set to test against.